Embedded System L1/AVR ATMEGA32/SPI MCU DRIVER/SPI/SPI/01-MCAL/04-SPI/spi.c

```diff
@@ -0,0 +1,245 @@
+/******************************************************************************************************
+ * File: spi.c 
+ * Author: Khalid Mosaad Zakaria
+ * Data: 14 july 2021
+ * Description: This File include APIs List.
+*******************************************************************************************************/
+#include"../../00-LIB/BIT_Math.h"
+#include"../../00-LIB/types.h"
+#include"spi.h"
+#include"spi_reg.h"
+#include"spiConfig.h"
+#include"../01-DIO/DIO_interface.h"
+
+
+#define NULL '\0'
+static void(*ptrf)(void) = NULL;
+
+uint8_t SPCR_Help;
+
+void SPI_DeInit(ST_SPI_config_t * SPI_InitStruct)
```

Comment:
?

Reply:
It does nothing i just forgot to delete and i did already. 
Thanks for your feedback.